src/modules/common/components/RemoteContent.js

```diff
@@ -73,25 +72,24 @@ class RemoteContent extends React.Component<PropType, StateType> {
     const width = this.state.webViewWidth
     const dataDetectorTypes: DataDetectorTypes = 'all'
     return <StyledView onLayout={this.onLayout}>
-      {// $FlowFixMe dataDetectorTypes (correct types, but Flow doesn't try the right branch)
-        <WebView
-          source={createHtmlSource(renderHtml(content, files, theme, RTL_LANGUAGES.includes(language) ? 'rtl' : 'ltr'),
-            URL_PREFIX + getResourceCacheFilesDirPath(cityCode))}
-          allowFileAccess // Needed by android to access file:// urls
-          originWhitelist={['*']} // Needed by iOS to load the initial html
-          javaScriptEnabled
+      <WebView
```

Comment:
The WebView has some breaking changes with the props (I upgraded it in NATIVE-9)